### ISSUE TITLE
Quote notebook install instructions in README

### ISSUE THREAD

**Issue by ashiklom** [openscm/openscm-twolayermodel#16]
`bash` doesn't seem to mind hard brackets, but `zsh` (note: the default shell on recent MacOS) doesn't like them in unquoted form and throws the following error:

```
zsh: no matches found: openscm-twolayermodel[notebooks]
```

This is easily resolved by quoting the full package name:

```
pip install "openscm-twolayermodel[notebooks]"
```

I would suggest tweaking this in the installation docs.

(Discovered as part of https://github.com/openjournals/joss-reviews/issues/2766).

**Comment by znicholls** [comment]
Awesome thanks! I've put a fix in #21 if you want to take a look

**Comment by ashiklom** [comment]
Looks great 👍
